spec/rubocop/cop/ezcater/rspec_dot_not_self_dot_spec.rb

```diff
@@ -45,9 +45,21 @@
       inspect_source(source)
       expect(cop.offenses).to be_empty
     end
+
+    it "accepts #{name} with `self.` not at the start of the string" do
+      source = "#{name} \"something self.class_method\" do\nend"
+      inspect_source(source)
+      expect(cop.offenses).to be_empty
+    end
+
+    it "accepts #{name} with `::` not at the start of the string" do
+      source = "#{name} \"something ::class_method\" do\nend"
+      inspect_source(source)
+      expect(cop.offenses).to be_empty
+    end
   end
 
-  RuboCop::RSpec::Language::Examples::ALL.each do |name|
+  described_class::EXAMPLE_IDENTIFIERS.each do |name|
```

Comment:
🎱  Similar to above, what's the rationale for changing this? Is there a reason to ignore `skip`, `pending`, and the `x`/`f`-prefix shortcuts?